packages/amazonq/.changes/next-release/Bug Fix-b3b09219-b76c-4892-90b4-45fada4a5e60.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Avoid refreshing code suggestion for paginated response"
```

Comment:
is there a way we can describe this that is meaningful to the customer

https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog